edx_solutions_projects/views.py

```diff
@@ -203,6 +203,19 @@ def workgroup_reviews(self, request, pk):
                 response_data.append(serializer.data)  # pylint: disable=E1101
         return Response(response_data, status=status.HTTP_200_OK)
 
+    @detail_route(methods=['get'])
+    def score(self, request, pk):
+        """
+        View final score for a specific Workgroup
+        """
+        content_id = self.request.query_params.get('content_id', None)
+        if not content_id:
+            return Response({'detail': 'Query string for "content_id" is required.'})
+        child_key = UsageKey.from_string(content_id)
```

Comment:
Since this is only supposed to work for specific types of content, perhaps you can add a quick check that childkey.category == 'gp-v2-project'